docs/csharp/language-reference/operators/snippets/Overview.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+
+namespace operators
+{
+    public static class Overview
+    {
+        public static void Examples()
+        {
+            // <SnippetExpressions>
+            int a, b, c;
+            a = 7;
+            b = a;
+            c = b++;
+            b = a + b * c;
+            a = (int)Math.Sqrt(b * b + c * c);
+            c = a >= 100 ? b : c / 10;
+            
+            string s;
+            char l;
+            s = "String literal";
+            a = s.Length;
+            l = s[3];
+            
+            List<int> numbers;
+            numbers = new List<int>(new[] { 1, 2, 3 });
```

Comment:
Unless there is a reason you have in mind for having them as two separate statements, I'd combine them.

```suggestion
            var numbers = new List<int>(new[] { 1, 2, 3 });
```

Reply:
I didn't want to mix declarations and assignments, but I think now that actually would be fine.